src/nedap/utils/spec/impl/check.cljc

```diff
@@ -13,6 +14,20 @@
   #?(:cljs 0
      :clj (Long/parseLong (System/getProperty "nedap.utils.spec.print-stack-frames" "0"))))
 
+(defmulti reporterm identity)
+
+(defmethod reporterm :default [_] `println)
+
+(defmethod reporterm :no-op [_] `comment)
+
+(defn reporter
+  "Returns the fully qualified symbol to used to report a spec failure.
+
+  The reporter is to be used as a fallback if the exception may be lost."
+  []
+  (reporterm #?(:cljs :default
+                :clj  (keyword (System/getProperty "nedap.utils.spec.printer" "default")))))
```

Comment:
At this point do we need the indirection of a multimethod? It could be a straight sysprop -> symbol conversion